ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -1093,6 +1093,19 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi-private
+- as: aws-ipi-private-disasterrecovery-fips-f14
```

Comment:
Suggest the name to `aws-ipi-private-fips-f14-disasterrecovery`.
The name style is `${workflow-name-without-prefix-cucushift-installer-rehearse}(-fips)?-fN-testtype`, existing examples are `xxx-fN-ui, xxx-fN-cloud`

Reply:
Just done some update, is it expected? @liangxia

Reply:
Just a reminder, there are many **disasterrecovery** tests which name is like **aws-ipi-private-disasterrecovery-fips-f14**, like aws-ipi-private-disasterrecovery-f360 in https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml. If you put disasterrecovery after fN will cause the cases name not consistent.

Reply:
We have several jobs added already at the very beginning when we first adapt to Prow CI, before we have the rules for test names. I'd suggest for new jobs, all of us follow the test name rules to have the style consistent, while for old existing jobs, leave some time for sub-teams to decide whether/when they will update it or not.

Reply:
Agree